src/pages/guides/sdk/sdk_publish_events.md

```diff
@@ -4,10 +4,16 @@ title: Publish Events | Adobe I/O Events SDK
 
 # Publish events
 
-Event publishers can publish events to the event receiver using the ADobe I/O Events SDK. 
+Event publishers can publish events to the event receiver using the Adobe I/O Events SDK. 
 
 For information on installing and using the SDK, please begin by reading the [getting started guide](sdk_getting_started.md).
 
+<InlineAlert variant="help" slots="header, text1" />
+
+Data Residency:
+
+Please note that all custom events are stored in US East region. 
```

Comment:
Why US East?
Why don't we simply say United States.
```suggestion
Please note that all custom events are stored in the United States (US) region. 
```

Reply:
Agreed, that's better. I have made this change in all the three places.